### ISSUE TITLE
Screen reader does not work for Windows application

### ISSUE THREAD

**Issue by PeterNjeim** [flutter/flutter#94706]
## Steps to Reproduce

1. Compile an application for Windows (non-UWP)

**Expected results:** Windows Narrator will go through the widgets as it does on Android and iOS.

**Actual results:** Windows Narrator (and JAWS) says: "\<Application name\> window, flutter view, pane" and highlights the whole window (excluding the titlebar), and stops there.

```
[√] Flutter (Channel stable, 2.5.3, on Microsoft Windows [Version 10.0.22000.348], locale en-CA)
    • Flutter version 2.5.3 at C:\Users\<username>\Documents\source\repos\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (7 weeks ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[√] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at C:\Users\<username>\AppData\Local\Android\sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Visual Studio - develop for Windows (Visual Studio Community 2022 17.0.1)
    • Visual Studio at C:\Program Files\Microsoft Visual Studio\2022\Community
    • Visual Studio Community 2022 version 17.0.31912.275
    • Windows 10 SDK version 10.0.22000.0

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.62.3)
    • VS Code at C:\Users\<username>\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.29.0

[√] Connected device (1 available)
    • Windows (desktop) • windows • windows-x64 • Microsoft Windows [Version 10.0.22000.348]

• No issues found!
```

Note that I do not know if Windows desktop screen reader is actually supported. The wording in the documentation is strange to say the least. It starts off the sentence with "For mobile", and then proceeds to give instructions for both mobile and desktop. The documentation then continues by talking about the web browser, which is unrelated to this issue. So in reality, I don't know if this is even a bug or just a feature that's being worked on or in a backlog. Whatever the case is, I actually could not find any GitHub issue related to this, so I decided to open my own. Any guidance on this would be appreciated, thanks.

And yes I did modify Flutter to get Visual Studio 2022 working for it, but that's the only modification I made.

**Comment by darshankawar** [comment]
@PeterNjeim 
There's an open issue to add screen reader support, https://github.com/flutter/flutter/issues/77838
Please follow-up in it for further updates.
Closing this as duplicate.

**Comment by PeterNjeim** [comment]
Thank you. Since the term "screen reader" doesn't appear in that issue, I missed it. Regardless, thanks for letting me know.
